src/ios/CDVCapture.m

```diff
@@ -266,7 +267,18 @@ - (void)captureVideo:(CDVInvokedUrlCommand*)command
         // iOS 4.0
         if ([pickerController respondsToSelector:@selector(cameraCaptureMode)]) {
             pickerController.cameraCaptureMode = UIImagePickerControllerCameraCaptureModeVideo;
-            // pickerController.videoQuality = UIImagePickerControllerQualityTypeHigh;
+            switch((int)([quality doubleValue]*10)) {
+                case 0: 
+                    pickerController.videoQuality = UIImagePickerControllerQualityTypeLow;
+                    break;
+                case 10: 
+                    pickerController.videoQuality = UIImagePickerControllerQualityTypeHigh;
+                    break;
+                default:
+                    pickerController.videoQuality = UIImagePickerControllerQualityTypeMedium;
```

Comment:
This `default` case does not match the expected default value of `1` (here it would correspond to thee new value `0.5`).

See README info, previously for Android only, but shared between Android and iOS with this PR
> value of `1` ( the default )

Reply:
Great catch! I made the correction

Reply:
👍

Reply:
Is the CI broken? It looks like this is causing the error `cordova-paramedic, pollForResults error: Error: Cannot get the event cache: it doesn't exist in the app. Got this instead: null` and it looks like other pull requests have the exact same error. It does not seem related to the code. I could be reading this wrong.

Reply:
Yes, CI is currently defunct for plugins.

Reply:
Thanks for confirming! Let me know if there is anything else I need to change

Reply:
> Is the CI broken?

Yes. Paramedic uses some third-party remote testing tools which has stopped working. One of our PMC members is trying to find a resolution, but a solution has eluded us so far.